lib/kino/process.ex

```diff
@@ -763,7 +763,7 @@ defmodule Kino.Process do
       case process_info(pid, :registered_name) do
         {:registered_name, []} ->
           case get_label(pid) do
-            :undefined -> inspect(pid)
+            :undefined -> format_for_mermaid_graph_node(pid, id)
             process_label -> format_for_mermaid_graph_node(pid, process_label)
```

Comment:
For the origin process we currently show `:root`, which can be confusing, so I would skip it

```suggestion
            :undefined ->
              if idx == 0 do
                inspect(pid)
              else
                # Use worker/supervisor id as label
                format_for_mermaid_graph_node(pid, id)
              end

            process_label ->
              format_for_mermaid_graph_node(pid, process_label)
```